.github/workflows/node.js.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run tests
-        run: forge test -vvv
+        run: forge test -vvv --via-ir
```

Comment:
Curious why this flag is needed?

Reply:
This is to update the pipeline to add the new optimizer. Basically it avoids having to suffer from stack to deep issues. It automagically moves the stack elements to memory during compile time. It first goes into an intermediate representation (YUL) and does optimizations at this level, before proceeding to bytecode optimizations.